io.github.jordanl2.ModularCalculator.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.jordanl2.ModularCalculator</id>
+  <metadata_license>FSFAP</metadata_license>
```

Comment:
we prefer `CC0-1.0` if possible.

Reply:
also has this file been submitted upstream?

Reply:
No worries, committed fix.

Reply:
You mean commit this file to the app repo? I can do that, but why?

Reply:
application should maintain the appdata file (and install it)

note that this is not flathub specific either.

Reply:
Got it, yes the app data file is in the source repo, I was moving it to flathub but I'll keep it in the source repo instead.